.github/workflows/ci.yml

```diff
@@ -90,3 +97,23 @@ jobs:
           SCALA_SCALA_DIR: scala_scala
           DOTTY_DIR: dotty
         run: script/smoke_test.sh
+
+      - name: copy nvim-treesitter highlight queries
+        if: ${{ runner.os == 'Linux' }}
+        shell: bash
+        run: cp ./nvim_treesitter/queries/scala/highlights.scm ./queries/highlights.scm
+
+      - name: Check if highlight queries are out of sync with nvim-treesitter
+        if: ${{ runner.os == 'Linux' }}
+        uses: tj-actions/verify-changed-files@v13
+        id: verify-changed-files
+        with:
+          files: |
+            queries/highlights.scm
+
+      - name: Test quries if out of sync with nvim-treesitter
+        if: steps.verify-changed-files.outputs.files_changed == 'true'
+        run: |
+            echo "::warning file=queries/highlights.scm::Queries in this repo are out of sync with nvim-treesitter"
+            git diff queries/highlights.scm
+            npm run test
```

Comment:
Pretty sure the answer is "yes", but for discussion sake: _should_ we necessarily fail the job if the queries are out of date?

PRs often change tree structure, and sometimes rename nodes - which breaks queries.

My personal opinion is that we are at a point (unless there's some catastrophic parser mistake current grammar has) where we can stabilise and treat breaking changes with a lot more care.

What are you thoughts @ckipp01 and @eed3si9n ?

Reply:
> My personal opinion is that we are at a point (unless there's some catastrophic parser mistake current grammar has) where we can stabilise and treat breaking changes with a lot more care.

I agree, except I'd say stabilize. If a PR breaks existing queries we should try working around it by creating a dummy node, or at least know that we are in that territory.

Reply:
> except I'd say stabilize

![image](https://user-images.githubusercontent.com/107868858/213217421-3ee0d0cf-ccd9-474b-a9c4-0e2fac0b9159.png)